src/intl/messages/es.json

```
@@ -292,7 +292,119 @@
 			"oldEmail": "Correo electrónico anterior",
 			"newEmail": "Nuevo correo electrónico",
 			"type": "Tipo",
+			"general": "General",
+			"notifications": "Notificaciones",
 			"belowToConfirm": "abajo para confirmar",
+			"Notifications": {
+				"notificationChannels": "Canales de Notificación",
+				"configureNotificationChannels": "Por favor seleccione las redes sociales donde le gustaría recibir notificaciones:",
+				"emailNotifications": "Notificaciones por Correo Electrónico",
```

Biome (2.1.2)

[error] The key notificationChannels was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)